camera/cameraDisplayModule.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+
+import os
+import robomodules as rm
+from messages import message_buffers, MsgType
+
+ADDRESS = os.environ.get("BIND_ADDRESS","localhost")
+PORT = os.environ.get("BIND_PORT", 11297)
+
+FREQUENCY = 10
+
+class CameraDisplayModule(rm.ProtoModule):
+    def __init__(self, addr, port):
+        self.subscriptions = [MsgType.CAMERA_MSG]
+        super().__init__(addr, port, message_buffers, MsgType, FREQUENCY, self.subscriptions)
+        self.value = -1
+        self.sub_ticks = 0
+        self.subbed = True
+
+    def msg_received(self, msg, msg_type):
+        # This gets called whenever any message is received
+        # We received pickled frames here.
+        if msg_type == MsgType.CAMERA_FRAME_MSG:
+            self.frame = msg.cameraFrame
+
+
+    def tick(self):
+        # this function will get called in a loop with FREQUENCY frequency
+        # process the serialized frame
+        displaySerializedImage(self.frame)
+
+    def displaySerializedImage(si):
```

Comment:
since this is a instance method, you should pass in self as the first parameter. then you can reference the frame using `self.frame`.

Reply:
Also, since this is a private method (should only be called from within the class, the name should be prefixed with `_`.
Finally, we follow the PEP8 styleguide, which means that method names and variable names should be in snake_case.
Eg.
```
def _display_serialized_image(self):
    frame = pickle.load(self.frame)
    ....
```

If you want to call it:
```
self._display_serialized_image()
```